### PR TITLE
Release

### PR DESCRIPTION
**Pull request by kuzdogan**
<sub>View in Huly <a href="https://app.huly.io/guest/w-kaanuzdogan-sourcify-65d30de0-75121c5c6c-fd615e?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJsaW5rSWQiOiI2NWU1ODFhNzhiNmVjZDA0ZDYyZDhkOTIiLCJndWVzdCI6InRydWUiLCJlbWFpbCI6IiNndWVzdEBoYy5lbmdpbmVlcmluZyIsIndvcmtzcGFjZSI6Incta2FhbnV6ZG9nYW4tc291cmNpZnktNjVkMzBkZTAtNzUxMjFjNWM2Yy1mZDYxNWUiLCJwcm9kdWN0SWQiOiIifQ.BeR3gPzz-iW_0TCTMT4fjBd72jtomqa3mElXtUyBVWg">HI-567</a></sub>

### CODE REVIEW

File: services/server/src/server/services/StorageService.ts [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  }
+
+  async init() {
+    try {
+      await this.ipfsRepository?.init();
+    } catch (e: any) {
+      throw new Error("Cannot initialize ipfsRepository: " + e.message);
+    }
+    try {
+      await this.allianceDatabase?.init();
```
Comment (kuzdogan) [inline services/server/src/server/services/StorageService.ts:49]: Should we execute init? I won't init no? @marcocastignoli

Comment (marcocastignoli) [reply services/server/src/server/services/StorageService.ts:49]: I would comment the `allianceDatabase.init()` call in any case, just to make it explicit that it's disabled

**Comment by kuzdogan** [comment]
We're set, can you do a final review on  these changes introduced after merging to staging: https://github.com/ethereum/sourcify/pull/1234/files/43b6a9e3eb5e7be73e3c8a5e9798debd4f605090..a5e30ff4d3c4ac4df07dead06b4c768104170706

@marcocastignoli

**Review by marcocastignoli — APPROVED** [review]
I gave it a quick look, and it looks fine. I cannot really tell much about the docker updates, but I trust you.

**Pull request merged**